eslint-rules.js

```diff
@@ -15,22 +15,28 @@ module.exports = {
     'import/extensions': 'off',
     'import/no-dynamic-require': 'off',
     'prefer-const': 'off',
+    'lines-between-class-members': 'off',
+    'template-curly-spacing': 'off',
+
+    indent: ['error', 2, {
+      ignoredNodes: ['TemplateLiteral']
+    }],
```

Comment:
Can you comment on why these were added?

Reply:
Woops, I think I blindly rushed through the lint error, my bad. We don't actually need it.